infrastructure/lambda/template.yaml

```diff
@@ -57,12 +57,20 @@ Conditions:
           - "none"
   AddJavaProvisionedConcurrency: !Not
     - !Equals
-      - !FindInMap [JavaProvisionedConcurrencyMapping, !Ref CriIdentifier, !Ref Environment]
-      -  0
+      - !FindInMap [
+          JavaProvisionedConcurrencyMapping,
+          !Ref CriIdentifier,
+          !Ref Environment,
+        ]
+      - 0
```

Comment:
Hmm what's  going on here? I haven't had these linting changes picked up on my branches?

Reply:
Hmm, good question - probably prettier VSCode plugin + save on format.

Have pushed again without the formatting changes :)